app/src/main/java/com/github/partymakers/partymaker/FragmentPartyMaker.java

```diff
@@ -0,0 +1,39 @@
+package com.github.partymakers.partymaker;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Button;
+
+
+import androidx.databinding.DataBindingUtil;
+import androidx.fragment.app.Fragment;
+
+import com.github.partymakers.partymaker.databinding.FragmentPartymakerBinding;
+import com.github.partymakers.partymaker.party.CreatePartyActivity;
+
+
+public class FragmentPartyMaker extends Fragment {
+    private FragmentPartymakerBinding dataBinding;
```

Comment:
Change class name to something more intuitive. Moving `Fragment` part to the end of the name, like with Activities, might be good too. I suggest `DashboardFragment`.